tenseal/proto/tensors.proto

```diff
@@ -18,3 +18,15 @@ message CKKSVectorProto {
     // Scale value
     double scale = 3;
 };
+
+//CKKSTensor parameters
+message CKKSTensorProto {
+    // The shape of the encrypted tensor
+    repeated uint32 shape = 1;
+    // The strides of the encrypted tensor
+    repeated uint32 strides = 2;
+    // The serialized ciphertext
+    repeated bytes ciphertext = 3;
```

Comment:
We will have a vector of ciphertexts here, I assume this declaration already support that, but maybe we can make it more explicit with `ciphertexts`

Reply:
"repeated" keyword is a vector for protobuffers.

I will update the name.